ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -3,8 +3,6 @@
 
 package com.daml.http
 
-import java.security.DigestInputStream
-import java.time.Instant
```

Comment:
Please revert "optimize imports"-related changes. Should be absolutely minimal with respect to import changes needed for this PR to compile.